DOL.WHD.Section14c.Domain/BaseEntity.cs

```diff
@@ -16,14 +16,33 @@ public class BaseEntity : IAuditedEntity
 
         [ForeignKey("LastModifiedBy_Id")]
         public ApplicationUser CreatedBy { get; set; }
-
-        public DateTime CreatedAt { get; set; }
+        
+        private DateTime? createdAt = null;
+        public DateTime CreatedAt
+        {
+            get
+            {
+                return createdAt ?? DateTime.Now;
+            }
+
+            set { this.createdAt = value; }
+        }
```

Comment:
Oh, I like this.  Provides a default value if one hasn't been set, right?

Reply:
Yes, it creates default value if empty.